data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/reconciler/impl/ResourcesReconcilerMessageHandler.java

```diff
@@ -67,17 +69,19 @@ private void reconcileLast(final DataPlaneContract.Contract newContract) {
       logger.info("Reconciling contract {}", keyValue("contractGeneration", contract.getGeneration()));
 
       resourcesReconciler.reconcile(contract.getResourcesList())
-        .onSuccess(v -> logger.info(
-          "Reconciled contract generation {}",
-          keyValue("contractGeneration", contract.getGeneration()))
-        )
-        .onFailure(cause -> logger.error(
-          "Failed to reconcile contract generation {}",
-          keyValue("contractGeneration", contract.getGeneration()),
-          cause
-          )
-        )
         .onComplete(r -> {
+          if (r.succeeded()) {
+            logger.info(
```

Comment:
nit: debug?

Reply:
This is useful to get an understanding of how far we're at reconciling resources, it's not on the hot path and it's not very verbose.